app/assets/javascripts/host_edit.js

```diff
@@ -76,6 +76,7 @@ function computeResourceSelected(item){
         if ($('#compute_resource').find('.alert-danger').length > 0) $('#compute_resource_tab a').addClass('tab-error');
         update_capabilities($('#capabilities').val());
         tfm.initByteSpinner();
+        tfm.initCounterSpinner();
```

Comment:
It seems like these two always come together, maybe have an `initFieldSpinners` function that runs both (and any more if needed in the future) instead of repeating?